sdk-springboot/src/main/java/io/dapr/springboot/DaprTopicSubscription.java

```diff
@@ -15,19 +15,23 @@
   private final String pubsubName;
   private final String topic;
   private final String route;
+  private final DaprTopicRoutes routes;
   private final Map<String, String> metadata;
 
   /**
    * Create a subscription topic.
    * @param pubsubName The pubsub name to subscribe to.
    * @param topic The topic to subscribe to.
    * @param route Destination route for messages.
+   * @param routes Destination routes with rules for messages.
    * @param metadata Metdata for extended subscription functionality.
    */
-  public DaprTopicSubscription(String pubsubName, String topic, String route, Map<String, String> metadata) {
```

Comment:
This is a breaking change. Please, keep the original constructor and add a new one instead.

Reply:
Done.